climada/util/test/test_forecast.py

```diff
@@ -50,3 +50,40 @@ def test_forecast_init():
     forecast = Forecast(lead_time=lead_times_seconds, member=[1, 2, 3])
     npt.assert_array_equal(forecast.lead_time, lead_times_seconds, strict=True)
     assert forecast.lead_time.dtype == np.dtype("timedelta64[ns]")
+
+
+def test_idx_member():
+    """Test idx_member method of Forecast class."""
+    forecast = Forecast(member=np.array([1, 2, 3, 4]))
```

Comment:
What happens if I put  `forecast = Forecast(member=np.array([1, 1, 3, -5, '6']))` ?

Reply:
Added a test for unusual data